src/transformKeysToSnake.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const isPlainObject = require('./isPlainObject')
+const snakeCase = require('./snakeCase')
+
+/**
+ * Transform object keys to snake case
+ * e.g. { fooBar: 1 } => { foo_bar: 1 }
+ * @param {Object} obj
+ * @returns {Object}
+ */
+const transformKeysToSnakeCase = (obj) => {
```

Comment:
should we have a `deep`/`recursive` flag to inform that nested objects should also be applied?

Reply:
What do you propose, the following?
```js
const transformKeysToSnakeCase = (obj, { recursive } = {}) => {
  if (Array.isArray(obj)) {
    return obj.map(item => transformKeysToSnakeCase(item))
  }

  if (!isPlainObject(obj)) {
    return obj
  }

  return Object.fromEntries(
    Object.entries(obj).map(([k, v]) => [
      snakeCase(k),
      recursive ? transformKeysToSnakeCase(v) : v
    ])
  )
}
```

Reply:
yeah, I give the consumer more control